deps/v8/src/log.cc

```diff
@@ -891,11 +892,13 @@ void Logger::ApiSecurityCheck() {
 
 void Logger::SharedLibraryEvent(const std::string& library_path,
                                 uintptr_t start,
-                                uintptr_t end) {
+                                uintptr_t end,
+                                uintptr_t slide) {
   if (!log_->IsEnabled() || !FLAG_prof_cpp) return;
   Log::MessageBuilder msg(log_);
-  msg.Append("shared-library,\"%s\",0x%08" V8PRIxPTR ",0x%08" V8PRIxPTR,
-             library_path.c_str(), start, end);
+  msg.Append("shared-library,\"%s\",0x%08" V8PRIxPTR ",0x%08" V8PRIxPTR
+                 ",0x%08" V8PRIxPTR,
```

Comment:
You should probably print the slide as a signed base 10 number because it can be < 0.  For that matter, the code should really be updated to use `intptr_t` because that's what `_dyld_get_image_vmaddr_slide()` returns.